packages/eslint-config/eslint-config.json

```diff
@@ -47,7 +47,7 @@
       {
         "devDependencies": [
           "rollup.config.js",
-          "test/**/*.js",
+          "*test/**/*.js",
```

Comment:
Was this intentional?

Reply:
Aye. Probably should generalize to `*test*` too, to catch test262.

```
❯ find packages -name '*?test' -depth 2
packages/eventual-send/integration-test
```